projects/br_scs/src/Mirza/BusinessRegistry/GenerateUtils.hs

```diff
@@ -22,16 +19,11 @@ dummyBusiness unique = NewBusiness (GS1CompanyPrefix ("Business" <> unique <> "P
                                    (mockURI unique)
 
 
-dummyUser :: T.Text -> GS1CompanyPrefix -> IO NewUser
-dummyUser unique business_uid = do
+dummyUser :: T.Text -> IO NewUser
```

Comment:
Does it need to be an `IO NewUser` anymore?
```suggestion
dummyUser :: T.Text -> NewUser
```

Reply:
Thanks for picking this up, fb44770d5e31301015d4f39b75c6507af141b209.